### PR TITLE
[server] Temp. disable expensive telemetry queries

### PR DESCRIPTION
**Pull request by geropl**
## Description
This change disables the execution of all telemetry queries. As discussed with @corneliusludmann  this it's meant as a temporary safety measure until a fix (#8638) is merged.

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Documentation

 - [x] /werft no-preview=true

### CODE REVIEW

**Comment by geropl** [comment]
/hold

Don't merge

File: components/server/src/installation-admin/telemetry-data-provider.ts [hunk 1]
```diff
@@ -18,9 +18,9 @@
      async getTelemetryData(): Promise<TelemetryData> {
             const data: TelemetryData = {
                 installationAdmin: await this.installationAdminDb.getData(),
-                totalUsers: await this.userDb.getUserCount(true),
-                totalWorkspaces: await this.workspaceDb.getWorkspaceCount(),
-                totalInstances: await this.workspaceDb.getInstanceCount(),
+                totalUsers: 0,  //await this.userDb.getUserCount(true),
```
Comment (jankeromnes) [inline components/server/src/installation-admin/telemetry-data-provider.ts:21]: Why disable the user count query? (This seems unrelated to an index on workspace.type)

Comment (geropl) [reply components/server/src/installation-admin/telemetry-data-provider.ts:21]: It was Saturday morning - and retrospectively, I would argue to be consistent :wink:
